paperpress.js

```diff
@@ -117,13 +118,40 @@ Paperpress.prototype._fileToItem = function (file) {
 
 	var item = {
 		title: name,
-		slug: slug
+		slug: slug,
+		content: fileContent,
+		path: null
 	}
 
-	item.path = this.pathBuilder(item, file.collectionName)
-	item.content = (fileType === '.md') ? this._marked.render(fileContent) : fileContent
+	if (frontMatter.test(fileContent)) {
+		var contentData = frontMatter(fileContent)
+		var dataAttributes = contentData.attributes
+
+		// Iterate through attributes and merge into the default item
+		for (var k in dataAttributes) {
+			if (dataAttributes.hasOwnProperty(k)) {
```

Comment:
@kinduff You're right, It's better to allow to add extra parameters.

How about this? This way we exclude the else path and we check the attribute leaking.

```js
Object.keys(dataAttributes)
  .filter(function (k) {
    return dataAttributes.hasOwnProperty(k)
  })
  .forEach(function (k) {
    item[k] = dataAttributes[k]
  })
```